ingestion/src/metadata/ingestion/ometa/mixins/es_mixin.py

```diff
@@ -54,7 +54,7 @@ def _search_es_entity(
         self,
         entity_type: Type[T],
         query_string: str,
-        fields: Optional[List[str]] = None,
+        fields: Optional[str] = None,
```

Comment:
Why we don't want to us a list? Easier to work with right?

Reply:
Yes lists are easier to work with but since we are using LRU cache it fails to store the list in the LRU cache hashmap

Reply:
ah damn